src/components/Styles.js

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components';
+  
+export const Button = styled.div`
+    position: fixed; /*Position is fixed regardless of scroll*/
+    left: 90%; /*Position is fixed at 90% from the left*/
+    bottom: 5px; /*Position is 5px from the bottom*/
+    font-size: 3rem; /*Size of button is at 3rem, rem is a unit relative to the root element*/
+    color: orange; /*Colour is orange*/
+`
```

Comment:
Do add a white fill for the up-arrow.
![image](https://user-images.githubusercontent.com/66960654/134700112-095b37c3-416e-4381-8ba2-c6efb7dc631a.png)